packages/vite/src/node/config.ts

```diff
@@ -285,6 +294,7 @@ export async function resolveConfig(
   defaultMode = 'development'
 ): Promise<ResolvedConfig> {
   let config = inlineConfig
+  config.isSPA ??= true
```

Comment:
it seems like the other config defaults are specified around line 480. is there a reason to do this one here instead of down there?

Reply:
👍 & done. (It also makes a functional difference: the default is set after the `config()` hooks are executed which is more consistent with the rest of the code. Good catch, thanks 👌.)